index.html

```diff
@@ -33,7 +32,7 @@
             background-color: rgba(255, 255, 255, 0.5);
             transition-timing-function: ease-in-out;
             transition-timing-function: cubic-bezier(0.42, 0, 0.58, 1);
-            transition: 0.2s;
+            transition: 5s;
```

Comment:
This transition speed doesn't look right, these things needs to transition in the 200-300ms range to feel responsive. Thjis one in particular was based based on the Material Design system (in particular the cubic bezier easing is one of the standard ones the library has/had): https://material.io/design/motion/speed.html